staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -211,6 +201,21 @@ func NewBadRequest(reason string) *StatusError {
 	}}
 }
 
+// NewTooManyRequests creates an error that indicates that the client must try again later because
+// the specified endpoint is not accepting requests. More specific details should be provided
+// if client should know why the failure was limited4.
+func NewTooManyRequests(message string, retryAfterSeconds int) *StatusError {
+	return &StatusError{metav1.Status{
+		Status:  metav1.StatusFailure,
+		Code:    http.StatusServiceUnavailable,
```

Comment:
Um, yes, I don't know how this didn't get copied.  I must have autocompleted the wrong field.  Terrifying.